src/app/shared/services/switchboard-toaster.service.ts

```diff
@@ -0,0 +1,76 @@
+import { Injectable } from '@angular/core';
+import { ToastrService } from 'ngx-toastr';
+import { IndividualConfig } from 'ngx-toastr/toastr/toastr-config';
+import { BehaviorSubject, Observable } from 'rxjs';
+
+enum MessageType {
+    error = 'toast-error',
+    info = 'toast-info',
+    success = 'toast-success',
+    warning = 'toast-warning'
+}
+
+export interface SwitchboardToaster {
+    message: string;
+    type: string;
+    isNew: boolean;
+}
+@Injectable({
+    providedIn: 'root'
+})
+export class SwitchboardToasterService {
```

Comment:
I'm not sure if this is a typo or on purpose, because this is wrapper for Toastr, not Toaster :) And for clarity I have nothing against this name! I'm just curious :D